dgraph/cmd/alpha/run.go

```diff
@@ -145,6 +145,8 @@ they form a Raft group and provide synchronous replication.
 		Flag("percentage",
 			"Cache percentages summing up to 100 for various caches (FORMAT: PostingListCache,"+
 				"PstoreBlockCache,PstoreIndexCache)").
+		Flag("keep-updates",
```

Comment:
delete-on-update

On true, we delete the key from cache when an update is made to the same key. Otherwise, we do an in place update in cache.

what difference does this parameter make to Dgraph?